drivers/vmwarevcloudair/vcloudair.go

```diff
@@ -407,66 +411,17 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if d.Provision {
-		dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
-
-		log.Infof("Installing Docker...")
-
-		cmd, err = d.GetSSHCommand(dockerInstall)
-		if err != nil {
-			return err
-		}
-
-		if err = cmd.Run(); err != nil {
-			return err
-		}
+	dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
```

Comment:
If you want to continue reaping the benefits of that script, I absolutely recommend you drop the `sudo` here (as @nathanleclaire says).  You should be aware however that the future of that script is cloudy (note that the last time it was touched was [Nov 9, 2014](https://github.com/docker/docker/commits/master/project/install.sh) when the directory was renamed, and [a PR on Oct 22, 2014 that was merged on Nov 4, 2014](https://github.com/docker/docker/commits/master/hack/install.sh) before that.  It's not heavily maintained specifically because platform detection is very _very_ fragile.

If we know the platform we're trying to install into, I definitely recommend more direct installation.  For example, in this case, I'm definitely very much +1 to performing the steps manually:
```bash
echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
apt-get update
apt-get install -y lxc-docker
```